ch_pipeline/processing/client.py

```diff
@@ -140,9 +140,10 @@ def create(type_):
 @click.argument("revision", type=PREV)
 @click.option("-l", "--long", is_flag=True)
 @click.option("-h", "--human", is_flag=True)
-def item_list(revision, long, human):
+@click.option("-t", "--time", is_flag=True)
+def item_list(revision, long, human, time):
     """List existing items within the REVISION (given as type:revision)."""
-    for tag in revision.ls():
+    for tag in revision.ls(time):
```

Comment:
The meaning of this option isn't clear. So at the least it should have a `help=` string. If you're feeling awesome you could add them for the options above too.